From 7677421be6e9bd8d4acac1b82294713a70cbc2e0 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Mon, 4 May 2015 10:56:59 +0800
Subject: [PATCH 0142/5242] MLK-9618-6 usb: chipidea: otg: show message for b
 host HNP fails

commit  238946c6f4e293d970cdbbe5db9ccfb5ca06d2c5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Show 'Device no response' message if A-device fails to connect as a peripheral.

Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit 134a56ed85141acfa86ba0d827f654f7716405f2)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/usb/chipidea/otg_fsm.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/chipidea/otg_fsm.c b/drivers/usb/chipidea/otg_fsm.c
index 3a24585..a5a81f6 100644
--- a/drivers/usb/chipidea/otg_fsm.c
+++ b/drivers/usb/chipidea/otg_fsm.c
@@ -317,6 +317,7 @@ static int a_aidl_bdis_tmout(struct ci_hdrc *ci)
 static int b_ase0_brst_tmout(struct ci_hdrc *ci)
 {
 	ci->fsm.b_ase0_brst_tmout = 1;
+	dev_warn(ci->dev, "Device No Response\n");
 	return 0;
 }
 
-- 
1.7.9.5

